scaffolds/rust/Cargo.lock

```diff
@@ -0,0 +1,4 @@
+[root]
```

Comment:
In our case, I wouldn't commit the `Cargo.lock` as they are scaffolding, i.e. piece of code that will be reused in other context.